root/main/timeout.tt

```diff
@@ -0,0 +1,8 @@
+[% WRAPPER "layout.tt" title=l('Request Timed Out') full_width=1 %]
+    <div id="content">
+        <h1>[% l('Request Timed Out') %]</h1>
+
+        <p><strong>[% l('Processing your request took too long and timed out.') %]</strong></p>
+        <p>[% l('It may help to try again by reloading the page.') %]</p>
+    </div>
```

Comment:
Feel free to comment on the language, of course.

I was also wondering whether we should include the strack trace here, perhaps in an element that is `display: none`? It may still be useful for developers without access to the log.